src/Collection.ts

```diff
@@ -311,13 +311,16 @@ export function createCollectionMixin<TBase extends Constructor>(Base: TBase) {
      * Given a bounding box, return all the objects of the collection that are contained in the bounding box.
      * If `includeIntersecting` is true, return also the objects that intersect the bounding box as well.
      * This is meant to work with selection. Is not a generic method.
+     *     /**
+     * Moves an object to specified level in stack of drawn objects
+     * @param {TBBox} bbox a bounding box in scene coordinates
+     * @param {{ includeIntersecting?: boolean }} options an object with includeIntersecting
```

Comment:
this is not very useful info but the above it so that should be moved here IMO

Reply:
Include objects that are contained in or intersect the bounding box

Reply:
I think that i wanted to copy a parameter section from another function and i pulled in a piece of comment without noticing. removing it.